src/kshell/cat.c

```diff
@@ -0,0 +1,36 @@
+#include "kshell.h"
+#include <fs/vfs.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+
+void cat(int argc, char* argv[])
```

Comment:
We probably want to include the prototype of this function in `kshell.h`. Thanks!

Reply:
Thanks for catching that.  I have added the function prototype to `kshell.h`.